### PR TITLE
DynamicRangeSlider fix

### PR DESCRIPTION
**Pull request by ShahAnuj2610**
- extending #988 
- fixes reset behaviour of DynamicRangeSlider

### CODE REVIEW

File: site/src/components/Home.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class HomePage extends Component {
 		// To fetch reactive search github stars
 		fetch('https://api.github.com/repos/appbaseio/reactivesearch')
 			.then(res => res.json())
-			.then(res => {
+			.then((res) => {
```
Comment (bietkul) [inline site/src/components/Home.js:68]: Why there are changes in this file?

**Review by bietkul — CHANGES_REQUESTED** [review]
@ShahAnuj2610 I'm not sure, what is exactly this PR doing.
I can see the changes in `vue` & `react` folder which is confusing.
alt text

File: site/src/components/Home.js (hunk 1 above)
Comment (ShahAnuj2610) [reply site/src/components/Home.js:68]: I back merged `next` and tried to commit, and it was throwing `eslint` errors for the above files, so had to fix it

File: packages/vue/src/components/range/DynamicRangeSlider.jsx [hunk 2]
```diff
@@ -1,12 +1,11 @@
 import VueTypes from 'vue-types';
+import VueSlider from 'vue-slider-component';
 import { Actions, helper } from '@appbaseio/reactivecore';
-import NoSSR from 'vue-no-ssr';
 import Container from '../../styles/Container';
```
Comment (bietkul) [inline packages/vue/src/components/range/DynamicRangeSlider.jsx:4]: You removed `vue-no-ssr`, Please make sure it works well with the server-side rendering.
Just test the `with-nuxt` example.
If it works then remove this package from `RangeSlider` component & `package.json`.

**Review by bietkul — CHANGES_REQUESTED** [review]

**Comment by bietkul** [comment]
@ShahAnuj2610 It lgtm, just make sure that it works well with SSR.

**Comment by bietkul** [comment]
Please also verify the issues commented by @mauxtin at [here](https://github.com/appbaseio/reactivesearch/pull/988#issuecomment-500521131).

**Comment by ShahAnuj2610** [comment]
@bietkul SSR and URLParams issues have been fixed

File: packages/vue/examples/with-nuxt/nuxt.config.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ module.exports = {
 	/*
 	 ** Plugins to load before mounting the App
 	 */
-	plugins: ['~/plugins/reactivesearch-vue'],
+	plugins: [{ src: '~/plugins/reactivesearch-vue', ssr: false }],
```
Comment (bietkul) [inline packages/vue/examples/with-nuxt/nuxt.config.js]: It doesn't solve the issue, it means that you're using ReactiveSearch without SSR.
We have to support SSR.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/vue/examples/with-nuxt/nuxt.config.js (hunk 3 above)
Comment (ShahAnuj2610) [reply packages/vue/examples/with-nuxt/nuxt.config.js]: https://github.com/NightCatSama/vue-slider-component/issues/114#issuecomment-353827607

Comment (bietkul) [reply packages/vue/examples/with-nuxt/nuxt.config.js]: I already know that but it is not gonna work.
Try some other thing, if doesn't work then don't upgrade the `vue-slider-component`.

Comment (bietkul) [reply packages/vue/examples/with-nuxt/nuxt.config.js]: @ShahAnuj2610 You can try it once https://nightcatsama.github.io/vue-slider-component/#/?hash=server-side-rendering-ssr

**Comment by ShahAnuj2610** [comment]
revert `vue-slider-component` to `2.8.2` for `SSR` to work, here is a working prototype of DRS:

drs_work

**Review by bietkul — APPROVED** [review]

**Pull request merged**